routes/review-routes.js

```diff
@@ -1,115 +0,0 @@
-const express = require("express");
-const router = express.Router();
-const Review = require("../models/review");
-const User = require("../models/user");
-const Company = require("../models/company");
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const sendTweet = require("../public/javascripts/twitter");
-
-// Route to render new review form. We get the companies from the DB so that the users can select the relevant one from a dropdown in the review form.
-
-router.get("/new", isLoggedIn, (req, res, next) => {
-  Company.find().then((companies) => {
-    res.render("reviews/newreview", { user: req.session.currentUser, companies, layout: false });
-  });
-});
-
-// Post route for submitting the review. The frontend form is dynamic - if a user signed up through Google and didn't submit thier companyName they'll be shown that on the form. If we receive that info, we want to update the user in the DB with it. Then we go on to create the review, and then attribute that review to the company it was left about.
-
-router.post("/new", isLoggedIn, (req, res, next) => {
-  let rev;
-  let filteredProBullets;
-  let filteredConBullets;
-  let comp;
-  const { companyBeingReviewed, overallScore, features, customerSupport, valueForMoney, easyToUse, distribution, proBullets, conBullets, reviewTitle } = req.body;
-  const { _id } = req.session.currentUser;
-  console.log('FILTERED BULLETS:', filteredProBullets)
-  if (!overallScore || !features || !customerSupport || !distribution || !valueForMoney || !easyToUse || !proBullets || !conBullets || !reviewTitle || !companyBeingReviewed) {
-    Company.find().then((companies) => {
-      return res.render("reviews/newreview", { user: req.session.currentUser, errorMessage: "Please complete all sections", companies, layout: false });
-    });
-    return;
-  } else {
-    filteredProBullets = proBullets.filter(bullet => bullet !== '')
-    filteredConBullets = conBullets.filter(bullet => bullet !== '')
-    if (req.body.companyName) {
-      console.log("req.body.com exists");
-      User.findByIdAndUpdate(_id, { companyName: req.body.companyName }, { new: true })
```

Comment:
I'd be a bit concerned in this whole flow about async actions. You're updating a user and also a review, it doesn't look like they're cross-dependent but they definitely have the potential to be and this is an anti-pattern. We should be conscious of issues we could possibly run into in the future. 

If it's completely necessary to do 2 db calls in the same route, make sure to use `async/await` or promises and chain them to avoid any async db issues